data/data-pipeline/data_pipeline/etl/sources/cdc_life_expectancy/etl.py

```diff
@@ -0,0 +1,69 @@
+from pathlib import Path
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger, download_file_from_url
+
+logger = get_module_logger(__name__)
+
+
+class CDCLifeExpectancy(ExtractTransformLoad):
+    def __init__(self):
+        self.FILE_URL: str = "https://ftp.cdc.gov/pub/Health_Statistics/NCHS/Datasets/NVSS/USALEEP/CSV/US_A.CSV"
```

Comment:
i just want to say lol that this file is named `US_A.CSV`. very important underscore there. ^.^

Reply:
lol i know!